src/components/StandardCard/StandardCard.jsx

```diff
@@ -144,6 +144,7 @@ const StandardCard = ({ card, setCards, ...props }) => {
           cardId={card._id}
           cardTags={card.tags}
           cardTitle={card.title}
+          cardDescription={card.description}
```

Comment:
when you merge your changes from main, just choose my changes and then coordinate from there. this is actually called criteria not description. My bad, i just should have merged earlier oops.